### ISSUE TITLE
Utility to download and install openshift installer overrides README file of ocs-ci repository

### ISSUE THREAD

**Issue by mbukatov** [red-hat-storage/ocs-ci#99]
## Description of the problem

Utility function which downloads and installs openshift installer also overrides README file of ocs-ci repository.

## Reproducer

1. clone ocs-ci repo
2. prepare all configuration (such as pull secret, aws config, see docs for full details)
3. run `python -m pytest -m deployment --cluster-name example-ocs-deployment`

## Actual Results

README file in root directory of ocs-ci contains README file of openshift installer instead of one file for ocs-ci:

```
[ocsqe@localhost ocs-ci]$ git status -sb
## local
 M README.md
[ocsqe@localhost ocs-ci]$ git diff
diff --git a/README.md b/README.md
index e3aa2c3..a2ca940 100644
--- a/README.md
+++ b/README.md
@@ -1,9 +1,18 @@
-# OCS-CI
+# OpenShift Install
 
-OCS CI is a framework to test Red Hat OCS features and functionality using AWS
-and other supported platforms. The framework is based on CephCI
-  ( https://github.com/red-hat-storage/cephci )
+The OpenShift installer `openshift-install` makes it easy to get a cluster
+running on the public cloud or your local infrastructure.
 
-Documentation is structured and available under docs directory
-inside this project. You can click [HERE](docs/README.md) to see
-the Table Of Content(TOC) of our documentation.
\ No newline at end of file
+To learn more about installing OpenShift, visit [docs.openshift.com](https://docs.openshift.com)
+and select the version of OpenShift you are using.
+
+## Installing the tools
+
+After extracting this archive, you can move the `openshift-install` binary
+to a location on your PATH such as `/usr/local/bin`, or keep it in a temporary
+directory and reference it via `./openshift-install`.
+
+## License
+
+OpenShift is licensed under the Apache Public License 2.0. The source code for this
+program is [located on github](https://github.com/openshift/installer).
[ocsqe@localhost ocs-ci]$
```

Relevant logs:

```
14:09:51 - MainThread - utility.utils - INFO - Downloading openshift installer
14:10:30 - MainThread - utility.utils - INFO - Executing command: tar xzvf openshift-install.tar.gz  
```

## Expected results

README file (and any other files tracked in ocs-ci repository) are not changed during the process.

It would be more reasonable to extract only the files we expect to use.

**Comment by dahorak** [comment]
This will be partially fixed by https://github.com/red-hat-storage/ocs-ci/pull/92 (the binaries are downloaded into `./bin` directory).

But extraction of only the files we expect to use is good idea, will think about it and implement it...

**Comment by dahorak** [comment]
@mbukatov fixed in 81a96b9.

**Comment by dahorak** [comment]
Fixed by https://github.com/red-hat-storage/ocs-ci/pull/92.
